pywren_ibm_cloud/compute/backends/docker/config.py

```diff
@@ -34,9 +36,18 @@ def load_config(config_data):
         config_data['docker'] = {'host': 'localhost'}
 
     if config_data['docker']['host'] not in ['127.0.0.1', 'localhost']:
-        if 'ssh_user' not in config_data['docker'] or 'ssh_password' not in config_data['docker']:
+
+        required_credentials_0 = ('ssh_user', 'ssh_password')
+        required_credentials_1 = ('ssh_user', 'ssh_key_filename', 'ssh_passphrase')
```

Comment:
@omerb01 this suppose to be optional. Not everyone need to provide ssh_key_filename and ssh_passphrase

Reply:
@gilv I disabled `ssh_passphrase` and made `ssh_key_filename` optional